README.md

```diff
@@ -147,6 +147,49 @@ matched), unless the `emailDomainMustHaveDot` option is disabled.
 Use `LinkType.EMAIL` for this, and see [test cases
 here](src/test/java/org/nibor/autolink/AutolinkEmailTest.java).
 
+### Hash tag extraction
```

Comment:
Can we use "hashtag" (without a space) everywhere? Seems to be the accepted way to write this: https://en.wikipedia.org/wiki/Hashtag

Also for code, please change `HashTag` to `Hashtag`.